index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+<body>
+  <h2 class="header">Hello Microverse!</h2>
```

Comment:
- [ ] According to the requirements, you should use the `h1` tag instead of the `h2` tag. 
> An H1 tag specifies the primary heading on your webpage while An H2 tag marks the first sub-heading after your document's main heading.

![image](https://user-images.githubusercontent.com/45231276/174905749-eef90e75-ddf7-4dac-967a-b9daa9c87b88.png)